source/2019-08-15-octane-release-plan.md

```diff
@@ -0,0 +1,252 @@
+
+### Visit the Octane Migration channel on Discord
+
+[Ember Discord](https://emberjs.com/community) has a new channel called `#topic-octane-migration` where you can ask any questions you have or get help along the way. If you are new to Discord, post in `#discord-server-admin` to request permissions to post.
+
+### Update your app to 3.12
+
+Update your app to 3.12 after it is released. [ember-cli-update](https://github.com/ember-cli/ember-cli-update) will help you out!
+
+### Brush up on classes
+
+Octane uses [Native JavaScript Classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), so we recommend that developers try them out with regular JavaScript examples to learn the basics, if they are not already familiar. Without some background knowledge of classes, it will be hard for developers to tell which pieces are specific to Ember versus native JavaScript syntax.
+
+### Consider trying Octane features that have already landed
+
+The rest of the sections below cover optional features in Octane that you can consider using now. These features are independent of each other. You can opt into them one at a time, in any order.
+
+You will be able to migrate to many of Octane's idioms automatically using codemods that we will finalize with Ember 3.14. We do not recommend trying to mass-migrate older code (like `@ember/component` Components) without those codemods.
+
+Even then, many teams may want to not convert older code at all, and just start using Octane idioms moving forward. **This is a valid, fully supported approach.** We will have a guide to upgrading describing the options and strategy that we will finalize with Ember 3.14. At the same time, all of the Ember Guides and Tutorials will be fully migrated to Octane idioms. You can get a sneak peek of those work-in-progress learning resources [here](https://emberjs.com/editions/octane)!
+
+### application-template-wrapper
+
+In classic Ember, your entire application is automatically wrapped in a `<div>` with the class `ember-application`. Octane applications do not automatically insert this unneccessary `<div>`.
+
+If your CSS (or JavaScript) relies on this `<div>`, you can explicitly add it to your `application.hbs` or refactor your code to no longer rely on it.
+
+You opt in to this change by
+
+```sh
+ember feature:disable application-template-wrapper
+```
+
+### jquery-integration
+
+In classic Ember, jQuery is automatically included, and `this.$` inside of component classes uses jQuery to select from the DOM. Ember Octane does not include jQuery automatically, nor does it include any direct jQuery integration in the component API.
+
+Ember's internals no longer depend on jQuery. If your app still uses jQuery directly and you want to continue using it, you can add `@ember/jquery` to your `package.json` and disable this classic feature. You will still be able to use all the other features of Octane.
```

Comment:
Disable or enable this classic feature?

Seems like what we're opting for here is "and keep using this classic feature"

Reply:
This feature has reverse boolean logic. It is unfortunate. Enabling/disabling does the opposite of what you would think.